applications/poisson/sine/input.json

```diff
@@ -20,5 +20,9 @@
         "OutputDirectory": "output/poisson/sine/",
         "OutputName": "test",
         "WriteOutput": "false"
+    },
+    "Input": {
```

Comment:
Input as a name is unspecific. The filename is input.json, so everything in this file is "input" from a user persective. Why not using "Grid" in accordance with the class `ExaDG::Grid` and the parameters in `GridData`?

Reply:
Yes, I also didn't like the naming. The name Grid sounds better.

Reply:
addressed in ce5731610269cac685066d2a69b823c059586cd3.